view/templates/widget_forumlist.tpl

```diff
@@ -2,12 +2,12 @@
 function showHideForumlist() {
 	if( $("li[id^='forum-widget-entry-extended-']").is(':visible')) {
 		$("li[id^='forum-widget-entry-extended-']").hide();
-		$("li#forum-widget-collapse").html(window.showMore);
```

Comment:
Is this global variable used anywhere else? If not, can you please remove its initialization?

Reply:
I didn't even knew that this had been some variable, nor do I know where it is defined :-)

Reply:
searching for this only resulted in finding just another place where it hadn't worked :-)

Reply:
How did this ever worked?

Reply:
Possibly because it was empty it never replaced the content and so the initial content was used?